src/main/java/com/pinterest/secor/io/impl/FlexibleDelimitedFileReaderWriterFactory.java

```diff
@@ -122,6 +122,7 @@ public void close() throws IOException {
     private final BufferedOutputStream mWriter;
     private Compressor mCompressor = null;
     private byte DELIMITER = getWriterDelimiter();
```

Comment:
One minor comment, the instance variable name 'DELIMITER' should not be all uppercase.  All uppercase is reserved for constant.  I think you can either do 'mDelimiter' or 'delimiter'